src/utils/extension/extensionCommunication.js

```diff
@@ -0,0 +1,19 @@
+function checkExtensionInstalled(setHasExtension) {
+  window.addEventListener("message", function (event) {
+    if (
+      event.source == window &&
+      event.data.message === "EXTENSION_CONFIRMATION_RESPONSE" &&
+      event.data.source === "ZEEGUU_INJECTED_CODE"
```

Comment:
ZEEGUU_READER_INJECTED_CODE? So it's clear that the code is injected by the extension?

Reply:
Done!

Reply:
Ok, I really apologize, I should have seen this before :( If above we call it `EXTENSION` and here we call it `ZEEGUU_READER` we're making our life harder for no reason. So can we make a final change to this and call it `EXTENSION_INJECTED_CODE`? If you want, I make the chance because I didn't pay attention at the beginning.

Reply:
No problem!

I will change it to `EXTENSION_INJECTED_CODE` then - I think using the extension is very clear, the reader requires one more step.